packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -249,6 +250,30 @@ export interface FieldProps {
 	 * @defaultValue If not specified, the key that is persisted is the property key that was specified in the schema.
 	 */
 	readonly key?: string;
+	/**
+	 * A default provider used for fields which were not provided any values.
+	 * @remarks
```

Comment:
```suggestion
	 * @privateRemarks
```
These are details that we wouldn't want an end customer to be reading.

Reply:
Updated to `@privateRemarks`